tests/general/CLOCBearTest.py

```diff
@@ -0,0 +1,91 @@
+import os
+from queue import Queue
+
+from coalib.settings.Section import Section
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.testing.BearTestHelper import generate_skip_decorator
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.results.Result import Result
+
+from bears.general.CLOCBear import CLOCBear
+
+
+def get_absolute_test_path(file):
+    return os.path.join(os.path.dirname(__file__),
+                        'cloc_test_files', file)
+
+
+def load_testfile(name):
+    return open(get_absolute_test_path(name)).readlines()
```

Comment:
I referred LicenseCheckBearTest.py for this. It is used without `with` statement. May be I'll open an newcomer issue for LicenseCheckBear too.